tests/jballerina-unit-test/src/test/resources/test-src/record/closed_record_type_inclusion.bal

```diff
@@ -286,9 +286,21 @@ type SimpleConstNegateExpr record {|
     "-" op = "-";
 |};
 
+type R0 record {
+    int|string f = 1;
+};
+
+type R1 record {
+    *R0;
+    string f;
+};
+
+
 function testTypeInclusionWithFiniteField() {
     SimpleConstNegateExpr expr = {};
+    R1 r1 = { f : "hello"};
     assertEquality(true, expr is UnaryExpr);
+    assertEquality("hello", r1.f);
 }
 
```

Comment:
Please add more tests.

E.g.,

```
type Foo record {|
    int a = 1250;
|};

type Bar record {|
    *Foo;
    byte a = 100;
|};

function testFn() {
    Bar b = {}; // Should use the default from `Bar`, not `Foo`.
}
```

Reply:
Was this added? Please reply to these comments with relevant commits and links to tests.

Reply:
Yes.
https://github.com/ballerina-platform/ballerina-lang/blame/3abc989bdd82a9a7d91a41d4a446c5efa1e9ed5c/tests/jballerina-unit-test/src/test/resources/test-src/record/open_record.bal#L696